src/DFTK.jl

```diff
@@ -149,4 +151,10 @@ export load_atoms
 include("external/etsf_nanoquanta.jl")
 include("external/abinit.jl")
 
+function __init__()
```

Comment:
Could you make a comment here explaining the mechanism?

Reply:
Yes, I probably should. Good point.